packages/app/src/stores/bookmark.ts

```diff
@@ -17,3 +20,19 @@ export const useSWRBookmarkInfo = (pageId: string | null | undefined): SWRRespon
     }),
   );
 };
+
+export const useSWRCurrentUserBookmarks = (page?: number| null| undefined): SWRResponse<IPageHasId[], Error> => {
```

Comment:
please add `x` like `useSWRxCurrentUserBookmarks`
Also you can use `Nullable<>` for the arg type.

Also, it's better to rename the `page` variable  to `pageNumber` or `pageNum`